src/package.json

```diff
@@ -28,8 +28,10 @@
     "@nestjs/core": "^9.0.0",
     "@nestjs/mapped-types": "^1.2.2",
     "@nestjs/platform-express": "^9.0.0",
+    "@nestjs/schedule": "^3.0.1",
     "@sentry/node": "^7.52.1",
     "atob": "^2.1.2",
+    "aws-sdk": "^2.1406.0",
```

Comment:
add @aws-sdk/client-rekognition

Reply:
i had didn't pushed package.json file that why now i had pushed package.json file also